spec/abci++/README.md

```diff
@@ -25,19 +25,15 @@ This allows Tendermint to run with applications written in many programming lang
 
 This specification is split as follows:
 
-- [Overview and basic concepts](./abci++_basic_concepts_002_draft.md) - interface's overview and concepts needed to understand other parts of this specification.
+- [Overview and basic concepts](./abci++_basic_concepts_002_draft.md) - interface's overview and concepts
+  needed to understand other parts of this specification.
 - [Methods](./abci++_methods_002_draft.md) - complete details on all ABCI++ methods
   and message types.
 - [Requirements for the Application](./abci++_app_requirements_002_draft.md) - formal requirements
   on the Application's logic to ensure liveness of Tendermint. These requirements define what
```

Comment:
Should we really mention "liveness" here? Do our target audience understand it?

Reply:
It's a spec. And the liveness implications of Tendermint are discussed extensively in that section and the "methods" one